Documentation/Books/Manual/Programs/Arangodump/Options.md

```diff
@@ -4,3 +4,26 @@ Arangodump Options
 Usage: `arangodump [<options>]`
 
 @startDocuBlock program_options_arangodump
+
+Encryption Options
```

Comment:
Yes, those options are not generated because are in the enterprise edition only. This is why I added a manual headline. The headline is level 2, as the others that are auto-generated (if I am not wrong): I believe it should be same level of Global Options, Log Options, Server Options, Ssl Options